docs/overview.md

```diff
@@ -0,0 +1,28 @@
+<!--[metadata]>
++++
+title = "Registry Overview"
+description = "High-level overview of the Registry"
+keywords = ["registry, on-prem, images, tags, repository, distribution"]
+[menu.main]
+parent="smn_registry"
++++
+<![end-metadata]-->
+
+# Overview of Docker Registry Documentation
+
+The Docker Registry documentation includes the following topics:
+
+* [Docker Registry Introduction](index.md)
+* [Understanding the Registry](introduction.md)
+* [Deploying a registry server](deploying.md)
+* [Registry Configuration Reference](configuration.md)
+* [Notifications](notifications.md)
```

Comment:
Below this point, these are advanced docs / specific cases that we probably don't want to push forward / show in the main menu, and that are linked from elsewhere.

The exceptions to that would be:

 * recipes.md (which does link to some of these)
 * help.md